docs/source/api/motor-board.rst

```diff
@@ -0,0 +1,94 @@
+Motor Board API
+===============
+
+The kit can control multiple motors simultaneously. One Motor Board can
+control up to two motors.
+
+Accessing the Motor Board
+-------------------------
+
+The motor board can be accessed using the ``motor_board`` property of
+the ``Robot`` object.
+
+.. code:: python
+
+   my_motor_board = r.motor_board
+
+This board object has an array containing the motors connected to it,
+which can be accessed as ``motors[0]`` and ``motors[1]``. The Motor Board is labelled so you know which
+motor is which.
+
+.. code:: python
+
+   r.motor_board.motors[0]
+   r.motor_board.motors[1]
+
+Powering motors
+---------------
+
+Motor power is controlled using `pulse-width modulation
+(PWM) <https://en.wikipedia.org/wiki/Pulse-width_modulation>`__. You set
+the power with a fractional value between ``-1`` and ``1`` inclusive,
+where ``1`` is maximum speed in one direction, ``-1`` is maximum speed
+in the other direction and ``0`` causes the motor to brake.
+
+.. code:: python
+
+   r.motor_board.motor[0].power  = 1
+   r.motor_board.motor[1].power = -1
+
+These values can also be read back:
+
+.. code:: python
+
+   r.motor_board.motor[0].power
+   >>> 1
+
+   r.motor_board.motor[1].power
+   >>> -1
+
+.. Warning:: Setting a value outside of the range ``-1`` to
```

Comment:
Perhaps mention that rapidly changing motor board power will anger it also?

Reply:
It looks like this is currently mentioned at the bottom of the Coast section, not sure but maybe it would fit better here

Reply:
https://github.com/sourcebots/sbot/pull/37/files/33357ce1c602844c545fabb09544aaed70dd213e#diff-2db30e1c9b545054bb05bff9e83238d5R90

Reply:
I agree that it would better to move the warning to this location.